src/EnergyPlus/ScheduleManager.hh

```diff
@@ -355,6 +461,10 @@ struct ScheduleManagerData : BaseGlobalStruct
     Array1D<ScheduleManager::WeekScheduleData> WeekSchedule; // Week Schedule Storage
     Array1D<ScheduleManager::ScheduleData> Schedule;         // Schedule Storage
 
+    // Schedule:File variables
+    std::vector<ScheduleManager::schedInputIdfPreprocessObject> allIdfSchedData;
+    std::vector<ScheduleManager::PreProcessedColumn> columnarData;
```

Comment:
A vector of PreProcessedColumn objects. This will hold the schedule data that will be later inserted into the schedule data structure.